src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -69,7 +70,15 @@ private protected override INameResolver CreateResolver(PowerFxConfig alternateC
         /// <inheritdoc/>
         protected override IExpression CreateEvaluator(CheckResult result)

```

Comment:
>CreateEvaluator [](http://example.com/codeflow?start=39&length=15)

nit: can this still share code with CreateEvaluatorDirect ?

Reply:
Would require a breaking change to CreateEvaluatorDirect. Do we wanna do that?